dashboard/lib/next-auth/dynamodb-adapter.ts

```diff
@@ -181,9 +183,9 @@ export async function updateUser(user: Partial<AdapterUser>) {
       ...(user.name ? {'#n': 'name'} : {}),
     },
     ExpressionAttributeValues: {
-      ...(user.email ? {'#e': 'email'} : {}),
-      ...(user.image ? {'#im': 'image'} : {}),
-      ...(user.name ? {'#n': 'name'} : {}),
+      ...(user.email ? {':e': user.email} : {}),
+      ...(user.image ? {':im': user.image} : {}),
+      ...(user.name ? {':n': user.name} : {}),
     },
```

Comment:
This was a bug from copying and pasting `ExpressionAttributeNames` from above. I corrected this to use values.